tests/Yaapii.Atoms.Tests/IO/ZipTest.cs

```diff
@@ -0,0 +1,66 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Xunit;
+using System.IO;
+
+namespace Yaapii.Atoms.IO.Tests
+{
+    public class ZipTest
+    {
+        [Fact]
+        public void ReturnsStreamGreater0()
+        {
+            string folderPath = Path.GetTempPath() + "\\" + "ZipTestFolder";
+            try
+            {
+                var folder = Directory.CreateDirectory(folderPath);
+                var newFile = File.Create(folder.FullName + "\\FileToZipOne.txt");
+                newFile.Close();
+                newFile = File.Create(folder.FullName + "\\FileToZipTwo.txt");
+                newFile.Close();
+                newFile = File.Create(folder.FullName + "\\FileToZipThree.txt");
+                newFile.Close();
+
+                var streamOfZipped = new Zip(folder);
+                Assert.InRange<long>(streamOfZipped.Stream().Length, 1, long.MaxValue);
+
+            }
+            catch (Exception)
+            {
+
+                throw;
+            }
+            finally
+            {
+                Directory.Delete(folderPath, true);
+            }
+        }
+
+        [Fact]
+        public void EntryExistsInStream()
+        {
+            string folderPath = Path.GetTempPath() + "\\" + "ZipTestFolder";
```

Comment:
You can use Path.GetDirectoryName("ZipTests") - that will result in a path which is inside the build directory (because build dir is the current working directory). 

Another Hint: If you must combine path parts like you do above, use Path.Combine. That way you do not need to add or remove slashes manually.

Reply:
Path.GetDirectoryName("ZipTests") returns an empty string. 
I can use Path.Combine(Directory.GetCurrentDirectory(), "ZipTest"); instead if it is unpreferred to use users temp folder

Reply:
Sorry, I meant Path.GetFullPath. Regatding MSDN, this should do the trick.